packages/api/src/utils/db-client.js

```diff
@@ -554,19 +563,66 @@ export class DBClient {
         'uploads_multipart_total',
       ])
 
-    if (error) {
-      throw new DBError(error)
-    }
+    const dagByteSizeQuery = metricsQuery
+      .select('name, dimensions, value, collected_at')
+      .match({
+        name: 'dagcargo_project_bytes_in_active_deals',
+        dimensions: '{{project,nft.storage}}',
+      })
+      .order('collected_at', { ascending: true })
+      .single()
 
-    if (!data || !data.length) {
-      return undefined
-    }
+    const d = new Date()
```

Comment:
More descriptive variable name here would be good!

```suggestion
    const weekAgo = new Date()
```

Reply:
fixed